src/Uno.UI.Runtime.Skia.Tizen/Tizen/TizenHost.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using ElmSharp;
+using SkiaSharp;
+using SkiaSharp.Views.Tizen;
+using Tizen.NUI;
+using Uno.Foundation.Extensibility;
+using Windows.Graphics.Display;
+using Windows.UI.Xaml;
+using WinUI = Windows.UI.Xaml;
+using WUX = Windows.UI.Xaml;
+using TizenWindow = ElmSharp.Window;
+using Tizen.Applications;
+using Uno.UI.Runtime.Skia.Tizen;
+
+namespace Uno.UI.Runtime.Skia
+{
+	public class TizenHost : ISkiaHost
+	{
+		[ThreadStatic]
+		private static TizenHost _current;
+
+		private readonly TizenApplication _tizenApplication;
+		private readonly Func<WinUI.Application> _appBuilder;
+		private readonly TizenWindow _window;
+		private readonly string[] _args;
+
+		public static TizenHost Current => _current;
+
+		/// <summary>
+		/// Creates a WpfHost element to host a Uno-Skia into a WPF application.
+		/// </summary>
+		/// <remarks>
+		/// If args are omitted, those from Environment.GetCommandLineArgs() will be used.
+		/// </remarks>
+		public TizenHost(Func<WinUI.Application> appBuilder, string[] args = null)
+		{
+			Elementary.Initialize();
+			Elementary.ThemeOverlay();
+
+			_current = this;
```

Comment:
Codacy found an issue: [Remove this assignment of '_current' or initialize it statically.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5993935)